LICENSE

```diff
@@ -295,151 +295,147 @@ Dual licensed under the MIT and GPL licenses.
-      permissions and limitations under the License.
+Copyright (c) 2006-2012 by OpenLayers Contributors
+Published under the 2-clause BSD license.
 
-    -------------------------------------------------------
-      Contains XMLHttpRequest.js
-      <https://code.google.com/p/xmlhttprequest/source/browse/trunk/source/XMLHttpRequest.js?r=29>
-      Copyright 2007 Sergey Ilinsky (http://www.ilinsky.com)
+Full license text below taken from https://github.com/openlayers/openlayers/blob/release-2.12/license.txt
+(Tag release-2.12 corresponded to commit 0412410be0ec43dc9a5c258b640a793fad41bd88 on May 16, 2015).
+
+    Copyright 2005-2012 OpenLayers Contributors. All rights reserved. See
+    authors.txt for full list.
+
+    Redistribution and use in source and binary forms, with or without modification,
+    are permitted provided that the following conditions are met:
+
+     1. Redistributions of source code must retain the above copyright notice, this
+    list of conditions and the following disclaimer.
+
+     2. Redistributions in binary form must reproduce the above copyright notice,
+    this list of conditions and the following disclaimer in the documentation and/or
+    other materials provided with the distribution.
+
+    THIS SOFTWARE IS PROVIDED BY OPENLAYERS CONTRIBUTORS ``AS IS'' AND ANY EXPRESS
+    OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+    MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+    SHALL COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+    LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+    PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+    LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+    ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+    The views and conclusions contained in the software and documentation are those
+    of the authors and should not be interpreted as representing official policies,
+    either expressed or implied, of OpenLayers Contributors.
+
+Includes compressed code under the following licenses:
 
-      Licensed under the Apache License, Version 2.0 (the "License");
-      you may not use this file except in compliance with the License.
-      You may obtain a copy of the License at
-      http://www.apache.org/licenses/LICENSE-2.0
+(For uncompressed versions of the code used please see the
+OpenLayers GitHub repository: <https://github.com/openlayers/openlayers/blob/release-2.12/>)
 
     -------------------------------------------------------
-      Contains portions of Gears <http://code.google.com/apis/gears/>
+        Contains XMLHttpRequest.js <http://code.google.com/p/xmlhttprequest/>
+        Copyright 2007 Sergey Ilinsky (http://www.ilinsky.com)
```

Comment:
Since XMLHTTPRequest.js is no longer released under the Apache license, Ross wants us to include a link to the XMLHTTPRequest release that shows the Apache license. I'm not clear *exactly* which version of XMLHTTPRequest.js this release of OpenLayers is using, but the last release that was under the Apache license, which was in 2007, was:

https://code.google.com/p/xmlhttprequest/source/browse/trunk/source/XMLHttpRequest.js?r=29

So we should include that link here, as we did previously (or a different release if you can figure out which exact release OpenLayers is using).